node/core/pvf/src/prepare/memory_stats.rs

```diff
@@ -247,4 +225,26 @@ mod getrusage {
 		}
 		Ok(result)
 	}
+
+	/// Gets the `ru_maxrss` for the current thread if the OS supports `getrusage`. Otherwise, just
+	/// returns `None`.
```

Comment:
This comment seems outdated `extract_max_rss_stat` returns an option, `get_max_rss_thread` returns `io::Result`